packages/mux-player/src/media-theme-mux/media-theme-mux.ts

```diff
@@ -75,7 +79,7 @@ export default class MediaThemeMux extends MediaTheme {
     };
 
     // if we don't have a src, we want to disable all hotkeys
-    props.nohotkeys = props.hasSrc ? props.nohotkeys : true;
+    props.nohotkeys = !props.hasSrc || props.isDialogOpen ? true : props.nohotkeys;
```

Comment:
nitpick(non-blocking): `props.nohotkeys || !props.hasSrc || props.isDialogOpen`

Reply:
In the case where a source isn't set, or the dialog is open, we want to override the user-provided value of `nohotkeys`.

Reply:
only if it's `false`, no?

Reply:
Ah, right, if nohotkeys is true, it probably doesn't matter what the others are set to. Was overthinking things.
Yeah, your suggestion is better.